{"name":"User repository find all failure","fullName":"TestUserFindAllSuite/User repository find all/TestFindAll_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727773563255,"stop":1727773563255,"uuid":"626803a6-7fd4-11ef-a987-ca15b1947338","historyId":"2f1f80d14e58f2aebc34998f4e50c9a3","testCaseId":"b5aff46d3879fec5f608fa8fbb7e3fbb","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"h19.net58.bmstu.ru"},{"name":"thread","value":"TestUserFindAllSuite/User repository find all/TestFindAll_Failure"},{"name":"suite","value":"User repository find all"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727773563255,"stop":1727773563255,"parameters":[{"name":"Error","value":"sql: no rows in result set: record does not exist"},{"name":"Target","value":"record does not exist"}]}]}